docs/organizations/invitations.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Invite users to your organization
+description: Learn how to invite users to your organization.
+---
+
+Organization invitations allow you to add new members to your organization, granting them access to organization-specific features and resources.
+
+Once you create an invitation, Clerk sends an email to the invited user with a unique invitation link. When the user visits the organization invitation link, they will be redirected to Clerk's [Account Portal sign-in page](/docs/customization/account-portal/overview#sign-in). If the user is already signed in, they will be redirected to your application's homepage (`/`). If you want to redirect the user to a specific page in your application, you can [specify a redirect URL when creating the invitation.](#redirect-url)
+
+## Create an invitation
+
+Clerk's [prebuilt components](/docs/components/overview) and [Account Portal pages](/docs/customization/account-portal/overview) handle all of the organization invitation flows, including creating invitations, managing them, and accepting them.
+
+However, if you want to build custom flows, see the following sections.
```

Comment:
I'm only mentioning this, because I have seen many people go for the Server-side approach, because they are using Server Actions and custom flows for example.
This implementation would be incorrect, because this is prone to the whole app gating rate limited, which will affect any of the application's end-users.

Reply:
so we should not be encouraging them to use the backend sdk to invite users to their app ?
we should only list that its an option to use a curl command if an application owner wants to generate an invitation

Reply:
Correct.

If the BAPI request is initiated from an action an end-user is taking we should advise against that.

If the BAPI request is initiated by developers of the app that don't have access to the end-user accounts, this should be allowed. It can definitely be a curl command.

Note: I understand this is frustrating, because nowadays people do a lot of stuff server side, so customers might expect this to simply work, but it doesn't. Just be proactive here, so we don't have issues were customers complain the whole app does not work, because they got rate limited.